SUBMITPR.md

```diff
@@ -0,0 +1,36 @@
+### Submitting a pull request
+
+Pull requests let you tell others about changes you've pushed to a repository on GitHub. Once a pull request is sent, interested parties can review the set of changes, discuss potential modifications, and even push follow-up commits if necessary. Everyone can submit pull request and they're useful in the shared repository model where they're used to initiate code review and general discussion about a set of changes before being merged into a mainline branch.
+
+This guide outlines the step by step instructions for submitting a pull request using the SourceTree desktop client application (http://www.sourcetreeapp.com/) 
+
+* Fork the repository by navigating to the repository on the Github website and clicking the "Fork" button
+
+![alt text](images/fork.PNG "Fork")
+
+
+* Clone a copy of the repository locally by clicking the "Clone in desktop" button
+
+![alt text](images/cloneInDesktop.PNG "Clone in desktop")
+
+* Before you do any changes, make sure that local master is up to date with the commits upstream. In order to do so you have to fetch the branches from the upstream repository, check out your fork's local master branch and then merge the changes from upstream/master into your local master branch. [Syncing a fork](https://help.github.com/articles/syncing-a-fork/) describes this process in more detail.
+
+
+* Create a new feature branch in SourceTree. When you create a new branch make sure that you select the master as the current branch 
+
+![alt text](images/createNewBranch.PNG "Create new branch")
+
+* Add a new code sample, and once you are satisfied with the addition, commit the change by clicking the "Commit" button in SourceTree. Before you commit the changes, you will have to provide a commit message that describes your changes.
+
+![alt text](images/commit.PNG "Commit")
+
+* Push the change to your fork. Click the "Push" button and make sure to select the correct repository and branch to push. You will want to push the changes to your fork and not to the upstream directly.
+
+![alt text](images/push.PNG "Push")
+
+* Use Github to create your Pull Request. Click the pull requests button and select "New pull request". The comparing changes page will open. You can click the button "compare across forks" and then select your fork and branch as shown in the picture below.
```

Comment:
* 'Click the pull requests button on the repository [homepage](https://github.com/esri/developer-support/) and select..'